### DISCUSSION TITLE
Sending an email to the complainant

### DISCUSSION THREAD

**Discussion by cesar-sanjuan** [globaleaks/globaleaks-whistleblowing-software#3469] [General]
Good morning
We are finalizing configuration testing prior to go-live.
We haven't changed the mail server yet to use our own because we wanted to test everything first.
Our processors ask us that a complainant can receive a copy of their complaint by email. Is it possible?
Thank you

**Comment by cesar-sanjuan** [comment]
Good morning.
We are about to launch the application in the production environment and the users responsible for managing complaints insist on knowing if communication by email is possible.
Thank you
greetings

**Comment by evilaliv3** [comment]
Thank you for your question.

No it is not possible because it would invalidate the security of the procedure.

The email is not recognized as a secure communication channel in the context of whistleblowing, is subject to intereption posisbilities and is not subjected to data retention policies.
